python_bot.py

```diff
@@ -19,9 +19,12 @@
 from threading import Timer
 import time
 import urllib.request
+import sys
 
 # local file imports
 import config
+sys.path.append('./commands/')
```

Comment:
can you not just call the method after the import help? do we have to touch the system path?

Reply:
Yea, it's something weird with the way python modules work, I guess? It only looks in the current working directory. Here's something information I found though that says otherwise. https://stackoverflow.com/questions/4142151/how-to-import-the-class-within-the-same-directory-or-sub-directory

I guess I'll need to try it other ways and see what works.

Reply:
I think that would be preferable to even temporarily messing with the system path if we don't have to.